lib/teiserver_web/live/account/relationship/index.ex

```diff
@@ -413,4 +475,21 @@ defmodule TeiserverWeb.Account.RelationshipLive.Index do
     |> assign(:ignores, ignores)
     |> assign(:blocks, blocks)
   end
+
+  def get_purge_days_cutoff(socket) do
+    duration = socket.assigns[:purge_cutoff]
+    [_, number, type] = Regex.run(~r/(\d) (month|year)/, duration)
+    {number, _} = Integer.parse(number)
+
+    cond do
+      type == "year" ->
+        number * 365
+
+      type == "month" ->
+        number * 365.0 / 12
+
+      true ->
+        raise("Incorrect value assigned to :purge_cutoff")
```

Comment:
The error should be received by the dev. The user can only choose options so would never get an error unless a future dev added an option that didn't work. So I probably need to make a test function to make sure a future dev doesn't input an option that would break.

Reply:
No, this is not really a dev here, because this value is coming from a user. Sure, using the web page as intended will not give you weird values but anyone could submit whatever they want.

Reply:
How could a user submit a value other than the options I have given them?